src/Eyes/Domain/DomainStatus.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace BeholderWebClient\Eyes\Domain;
+
+use BeholderWebClient\Eyes\Status;
+
+class DomainStatus extends Status {
+
+  const REQUERIMENT_FAIL = 'REQUERIMENT FAIL: whois command not found with type -P whois';
```

Comment:
I think you dont believe what as writen, the command Whois have been not found With the another command called type.